### ISSUE TITLE
What is image_score?

### ISSUE THREAD

**Issue by ssun-g** [VitjanZ/DRAEM#14]
What is `image_score`?
I don't understand what that means.

in `test_DRAEM.py` (line 107)
https://github.com/VitjanZ/DRAEM/blob/2dbf67397ab5c10a1494e5ae70ab59a25d7c35ef/test_DRAEM.py#L107

in addition, Can I get F1_score in test(evaluate)?

**Comment by ssun-g** [comment]
in addition, Can I get F1 score in test(evaluate)?

**Comment by ssun-g** [comment]
I found it.
https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_auc_score.html

> In the binary case, it corresponds to an array of shape (n_samples,). Both probability estimates and non-thresholded decision values can be provided. **The probability estimates correspond to the probability of the class with the greater label** i.e. `estimator.classes_[1]` and thus `estimator.predict_proba(X, y)[:, 1]`

---
**Probably, it means that probability of being abnormal in binary case.**


https://github.com/VitjanZ/DRAEM/blob/2dbf67397ab5c10a1494e5ae70ab59a25d7c35ef/test_DRAEM.py#L119
